### DISCUSSION TITLE
Welcome to kyuubi Discussions!

### DISCUSSION THREAD

**Discussion by yaooqinn** [apache/kyuubi#261] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by rvacaru** [comment]
Hi Kyuubi people!

I'm Razvan, a software/data engineer working with Spark and more. I have a background in computer science and worked as a java developer before switching to data engineering almost 2 years ago. I'm curious to knwo if you'll participate in [Google Summer of Code 2022](https://summerofcode.withgoogle.com/) because I'd love to try and contribute to Kyuubi.

Ps. I'm also a big fan of Naruto :)

**Comment by yaooqinn** [comment]
Welcome @rvacaru,

Thanks for being interesting in the Apache Kyuubi project. I'm sure that you can help a lot to our community and project with your great background.

Sadly, we haven't applied for GSoC 2022 yet. Is there any way to make it happen as I find the button turns gray now? It seems an excellent opportunity to grow our community.

However, you are welcome to become a Kyuubi contributor no matter whether we are in that program or not. We strictly follow the 'Apache Way' to build our community; any kind of help is welcome.

PS. welcome you again as a Naruto fan.

BR,

Kent Yao

**Reply by rvacaru** [reply]
Hey Kent Yao, 

I think it's still early for GSoC 2022 applications. Starting from this edition the program will be open also to non students who'd like to contribute to Open source, while before it was mainly for students.

Maybe I'm gonna try with a fist good issue or with some testing if I have some spare time during holidays :)

KR,
Razvan Vacaru

**Reply by yaooqinn** [reply]
Glad to know that. Looking forward to your first contribution. Enjoy your holidays!
